code.js

```diff
@@ -0,0 +1,115 @@
+const cost = 'Cur_OfficialRate';
+const abr = 'Cur_Abbreviation';
+const scale = 'Cur_Scale';
+const inp = 'in_';
+let hidden = true;
+const select = ' ✓';
+const con = 'con_';
+
+document.getElementById('currency_list').hidden = true;
```

Comment:
If it refers to a converter component, do it on init your component-class

Reply:
done